Documentation/hubble.rst

```diff
@@ -0,0 +1,100 @@
+****************
+Hubble internals
+****************
+
+.. note:: This documentation section is targeted at developers who are willing
```

Comment:
nit: maybe change it to: `who are interested in contributing to Hubble`

Reply:
Better indeed. Updated.